### PR TITLE
libclc 16.0.3 (new formula)

### PR DESCRIPTION
**Pull request by paperchalice**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/libclc.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+class Libclc < Formula
+  desc "Implementation of the library requirements of the OpenCL C programming language"
+  homepage "https://libclc.llvm.org/"
+  url "https://github.com/llvm/llvm-project/releases/download/llvmorg-16.0.3/libclc-16.0.3.src.tar.xz"
+  sha256 "b8f11158cd298062602285391496bb2acaf7219923fdd13b93927753c491e7c1"
+  license "Apache-2.0" => { with: "LLVM-exception" }
+
+  depends_on "cmake" => :build
+  depends_on "llvm" => [:build, :test]
```
Comment (SMillerDev) [inline Formula/libclc.rb:9]: Why is this needed?

**Review by carlocab — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@paperchalice, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!
